packages/components/src/Enumeration/Items/Item/Item.scss

```diff
@@ -82,6 +82,10 @@ $item-checkbox-size: 20px;
 		vertical-align: top;
 	}
 
+	button span svg {
+		vertical-align: sub;
```

Comment:
Instead of putting the icon in the same span as the text, you can put it as sibling, directly as button child.
The button is already styled to vertically align the content.
So no extra style needed